test/encoder/encoder_native_test.exs

```diff
@@ -8,7 +8,7 @@ defmodule Encoder.NativeTest do
     assert {:ok, file} = File.read(in_path)
     assert {:ok, ref} = Enc.create(320, 240, :I420, :fast, :high, -1, -1, 30, 1, 23)
     assert <<frame::bytes-size(115_200), _tail::binary>> = file
-    assert {:ok, _dts_list, _frames} = Enc.encode(frame, 0, false, ref)
-    assert {:ok, [_dts], [_frame]} = Enc.flush(false, ref)
+    assert {:ok, _dts_list, _pts_list, _frames} = Enc.encode(frame, 0, false, ref)
+    assert {:ok, [_dts], [_pts], [_frame]} = Enc.flush(false, ref)
```

Comment:
As discussed, it would be best to have at least a basic test actually checking the timestamps.

As a bare minimum I'd consider verification that your changes work, you get frames with valid PTSs